3rdparty/obiwarp/mat.cpp

```diff
@@ -53,11 +53,26 @@ MatF::MatF(const MatF &A) : _m(A._m), _n(A._n), _dat(A._dat) {
 #endif
 }
 
-std::vector<float> MatF::pointer(int m)
+std::vector<float> MatF::row(int m)
 {
     return _dat.slice((m * _n), (m * _n) + _n);
 }
 
+std::pair<std::vector<float>::iterator, std::vector<float>::iterator> MatF::rowIters(int m)
+{
+    return _dat.slice2((m * _n), (m * _n) + _n);
+}
+
+float* MatF::rowData(int m)
+{
+    return (_dat.data() + (m * _n));
+}
+
+std::vector<float>::iterator MatF::rowIter(int m)
```

Comment:
As far as I can see this is not being used. You can get the same result by `rowIters.first` anyway.

Reply:
Oh its not? I was using it at one point. I'll remove it now.